### ISSUE TITLE
No such component directory: .. streamlit_labelstudio/frontend/build

### ISSUE THREAD

**Issue by robmarkcole** [deneland/streamlit-labelstudio#1]
Hi @deneland 
probably due to my ignorance of label studio, but on following these steps:
```
python3 -m venv venv
source venv/bin/activate
pip install streamlit-labelstudio
venv/bin/streamlit run app.py
```

The app raises exception:
```
StreamlitAPIException: No such component directory: '/Users/robin/Github/streamlit-labelstudio/streamlit_labelstudio/frontend/build'

File "/Users/robin/Github/streamlit-labelstudio/app.py", line 2, in <module>
    from streamlit_labelstudio import st_labelstudio
File "/Users/robin/Github/streamlit-labelstudio/streamlit_labelstudio/__init__.py", line 38, in <module>
    _component_func = components.declare_component("streamlit_labelstudio", path=build_dir)
```

I guess this might be due to Label Studio not being running?

**Comment by deneland** [comment]
Hey @robmarkcole , thanks for the interest!

No it should be running like that, I'll try to replicate later today/tomorrow!

**Comment by pohaulin** [comment]
I have the same problem like @robmarkcole

**Comment by AlexandreKempf** [comment]
I have the same problem as @robmarkcole :)

**Comment by interactivetech** [comment]
It seems the label studio front end code comes from this repo: [https://github.com/heartexlabs/label-studio-frontend#development](https://github.com/heartexlabs/label-studio-frontend#development)

And the build instructions are defined in the [package.json](https://github.com/heartexlabs/label-studio-frontend/blob/c33852562a799147216871d56bfc4bc800da70b2/package.json)

build-source points to `npm run build`, so when I enter in the `streamlit-labelstudio/streamlit_labelstudio/frontend` directory, and run `npm run build`, the labelstudio front end code compiles, and you get the build directory that is needed to run the streamlit app.

So in summary, `npm run build` in the `streamlit-labelstudio/streamlit_labelstudio/frontend directory` resolved my ability to run the app!

**Comment by jonas-nothnagel** [comment]
same problem as the rest.

**Comment by saharudra** [comment]
> The solution above me works, but the app seems to not load fully and displays only a part. ![image](https://user-images.githubusercontent.com/41116329/114551071-92027380-9c6b-11eb-8e74-d77860699379.png)

Is there any solution for this issue?
